src/main/features/response/form/models/whenDidYouPay.ts

```diff
@@ -0,0 +1,54 @@
+import { IsDefined, MaxLength } from 'class-validator'
+import { IsPastDate } from 'forms/validation/validators/datePastConstraint'
+import { LocalDate } from 'forms/models/localDate'
+import { IsNotBlank } from 'app/forms/validation/validators/isBlank'
+import { IsValidYearFormat } from 'app/forms/validation/validators/isValidYearFormat'
+import { MomentFactory } from 'common/momentFactory'
+import { MomentFormatter } from 'app/utils/momentFormatter'
+import { ValidationConstraints } from 'forms/validation/validationConstraints'
+import { ValidationErrors as DefaultValidationErrors } from 'forms/validation/validationErrors'
+
+const currentDate = MomentFormatter.formatLongDate(MomentFactory.currentDate())
```

Comment:
This is not right, current date will be evaluated only once when application loaded this source file. It would basically mean that current date is set to application startup date.

FIX IT!